packages/components/src/mobile/inserter-button/index.native.js

```diff
@@ -66,6 +66,19 @@ class MenuItem extends Component {
 			item.title
 		);
 
+		const sparkles = (
+			<SVG
+				viewBox="0 0 24 24"
+				fill="none"
+				xmlns="http://www.w3.org/2000/svg"
+			>
+				<Path
+					d="M10 11c-1.588-.479-4-.91-4-.91s2-.241 4-.454c1.8-.191 3.365-.502 4-3.181C14.635 3.775 15 1 15 1s.365 2.775 1 5.455c.635 2.679 2 2.969 4 3.181 2 .213 4 .455 4 .455s-2.412.43-4 .909c-1.588.479-3 1-4 4.546-.746 2.643-.893 4.948-1 5.454-.107-.506-.167-2.5-1-5.454C13 12 11.588 11.479 10 11zM7.333 3.5C6.803 3.333 6 3.182 6 3.182s.667-.085 1.333-.16c.6-.066 1.122-.175 1.334-1.113C8.878.971 9 0 9 0s.122.971.333 1.91c.212.937.667 1.038 1.334 1.113.666.074 1.333.159 1.333.159s-.804.15-1.333.318c-.53.167-1 .35-1.334 1.59C9.085 6.017 9.036 6.824 9 7c-.036-.177-.056-.875-.333-1.91-.334-1.24-.804-1.423-1.334-1.59zM2.444 18C1.474 17.713 0 17.454 0 17.454s1.222-.145 2.444-.272c1.1-.115 2.057-.302 2.445-1.91C5.277 13.666 5.5 12 5.5 12s.223 1.665.611 3.273c.388 1.607 1.222 1.781 2.445 1.909 1.222.127 2.444.273 2.444.273s-1.474.258-2.444.545c-.971.287-1.834.6-2.445 2.727-.456 1.586-.546 2.97-.611 3.273-.065-.304-.102-1.5-.611-3.273C4.278 18.6 3.415 18.287 2.444 18z"
+					fill="#F0C930"
+				/>
+			</SVG>
+		);
+
```

Comment:
might be worth moving the sparkles icon module from the icons package into this package. sure it's fine having the SVG defined inline here, but it's obviously generated/optimized and so practically _un-editable_ in place and so a bit awkward in the middle of another component.

not a big issue and some may disagree with me on this, but I think keeping it an import from another file would make it easier to think about and maintain. we wouldn't have to call it sparkles even; we could give it a semantic name like `block-is-new-icon.svg` and then people would have the freedom to change it to something that isn't sparkles without changing the intent of the code/module/import.

Reply:
Thanks a bunch, that's much better. I took a stab at that in https://github.com/WordPress/gutenberg/pull/38849/commits/499d4dcb64b6824812fdf4149f95c00a9d65d6b3, hopefully I got the code right.